web-app/js/portal/common/helpers.js

```diff
@@ -316,3 +316,8 @@ function getPortalBase(pathname) {
 function normaliseLongitude(longitude) {
     return (longitude + 540) % 360 - 180;
 };
+
+// Remove all child nodes of HTML elements from the DOM
+function clearContents(jQuerySelector) {
```

Comment:
Since this is used only in `FacetedSearchResultsDataView.js` and is not necessarily very reusable - I'd stick it in `FacetedSearchResultsDataView.js`.